README.md

```diff
@@ -51,13 +51,13 @@ compile group: 'org.truemail', name: 'truemail-java-client', version: 'version'
 ## Usage
 
 ```java
-import org.truemail.Client;
-import org.truemail.client.Configuration;
+import org.truemail.TruemailClient;
+import org.truemail.client.TruemailConfiguration;
 ```
 
-To have an access for `Client#validate` you must create configuration instance first as in the example below:
+To have an access for `TruemailClient#validate` you must create configuration instance first as in the example below:
 
-### Creating configuration instance
+### Creating TruemailConfiguration instance
```

Comment:
Should be `### Creating configuration instance`, because it's topic title.

Reply:
Good catch, thanks!)